"panic: index out of range" with playlist command

Hi, I was just trying out your tool and ran into an issue when trying to play a folder of videos. Here's what I ran:

```
# start a docker container
docker run --name=cc -v /z:/z -it --network=bridge --rm golang:alpine sh
# in the container
go get -u github.com/vishen/go-chromecast
# get the uuid of my target chromecast
go-chromecast ls
# try to play a playlist
go-chromecast playlist /z/_cat_video/Paul_Dinning/ -u 93fe0c0a723e8381d61ca4de663fdaf6 --verbose -v
```
This results in the error shown here
```
using device name=Living Room Chromecast addr=192.168.0.223 port=8009 uuid=93fe0c0a723e8381d61ca4de663fdaf6
DEBU[0000] (1)sender-0 -> receiver-0 [urn:x-cast:com.google.cast.tp.connection]: {"type":"CONNECT","requestId":1}  package=cast
DEBU[0000] (2)sender-0 -> receiver-0 [urn:x-cast:com.google.cast.receiver]: {"type":"GET_STATUS","requestId":2}  package=cast
DEBU[0000] (2)sender-0 <- receiver-0 [urn:x-cast:com.google.cast.receiver]: {"requestId":2,"status":{"applications":[{"appId":"CC1AD845","appType":"WEB","displayName":"Default Media Receiver","iconUrl":"","isIdleScreen":false,"launchedFromCloud":false,"namespaces":[{"name":"urn:x-cast:com.google.cast.cac"},{"name":"urn:x-cast:com.google.cast.debugoverlay"},{"name":"urn:x-cast:com.google.cast.broadcast"},{"name":"urn:x-cast:com.google.cast.media"}],"sessionId":"4b667151-34f7-4f99-808e-09a140545921","statusText":"Default Media Receiver","transportId":"4b667151-34f7-4f99-808e-09a140545921","universalAppId":"CC1AD845"}],"userEq":{},"volume":{"controlType":"attenuation","level":0.44999998807907104,"muted":false,"stepInterval":0.05000000074505806}},"type":"RECEIVER_STATUS"}  package=cast
DEBU[0000] (3)sender-0 -> 4b667151-34f7-4f99-808e-09a140545921 [urn:x-cast:com.google.cast.tp.connection]: {"type":"CONNECT","requestId":3}  package=cast
DEBU[0000] (4)sender-0 -> 4b667151-34f7-4f99-808e-09a140545921 [urn:x-cast:com.google.cast.media]: {"type":"GET_STATUS","requestId":4}  package=cast
DEBU[0000] (4)sender-0 <- 4b667151-34f7-4f99-808e-09a140545921 [urn:x-cast:com.google.cast.media]: {"type":"MEDIA_STATUS","status":[],"requestId":4}  package=cast
panic: runtime error: index out of range [4] with length 4

goroutine 1 [running]:
github.com/vishen/go-chromecast/cmd.glob..func8.1(0x67, 0x66, 0x0)
        /go/pkg/mod/github.com/vishen/go-chromecast@v0.2.9/cmd/playlist.go:128 +0x125
sort.insertionSort_func(0xc0006bfbe0, 0xc000362060, 0x65, 0x6d)
        /usr/local/go/src/sort/zfuncversion.go:12 +0xab
sort.quickSort_func(0xc0006bfbe0, 0xc000362060, 0x65, 0x6d, 0xc)
        /usr/local/go/src/sort/zfuncversion.go:158 +0x1f6
sort.quickSort_func(0xc0006bfbe0, 0xc000362060, 0x65, 0x7c, 0xc)
        /usr/local/go/src/sort/zfuncversion.go:145 +0xf5
sort.quickSort_func(0xc0006bfbe0, 0xc000362060, 0x3f, 0x7c, 0xd)
        /usr/local/go/src/sort/zfuncversion.go:148 +0x131
sort.quickSort_func(0xc0006bfbe0, 0xc000362060, 0x0, 0x7c, 0xe)
        /usr/local/go/src/sort/zfuncversion.go:148 +0x131
sort.Slice(0xb4ad40, 0xc00000e030, 0xc0006bfbe0)
        /usr/local/go/src/sort/slice.go:20 +0xe8
github.com/vishen/go-chromecast/cmd.glob..func8(0x11ae220, 0xc0003ee000, 0x1, 0x5, 0x0, 0x0)
        /go/pkg/mod/github.com/vishen/go-chromecast@v0.2.9/cmd/playlist.go:108 +0x575
github.com/spf13/cobra.(*Command).execute(0x11ae220, 0xc00007df90, 0x5, 0x5, 0x11ae220, 0xc00007df90)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x11ac920, 0xed8773c21, 0x0, 0xc8ce1b)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/vishen/go-chromecast/cmd.Execute(0xc7c140, 0x3, 0xc7e5c0, 0x6, 0x0, 0x0, 0xc00003a0b8)
        /go/pkg/mod/github.com/vishen/go-chromecast@v0.2.9/cmd/root.go:60 +0xa7
main.main1(...)
        /go/pkg/mod/github.com/vishen/go-chromecast@v0.2.9/main.go:35
main.main()
        /go/pkg/mod/github.com/vishen/go-chromecast@v0.2.9/main.go:31 +0x69
```
The video files in question are a mix of mkv and mp4 files I downloaded from YouTube, and there's 249 of them in this folder. Here are the file names in case that helps.
https://pastebin.com/7mxDjtFJ

--- a/cmd/playlist.go
+++ b/cmd/playlist.go
@@ -119,10 +119,10 @@ that ffmpeg is installed.`,
 				max = len(jNum)
 			}
 			for vi := 0; vi < max; vi++ {
-				if len(iNum) < vi {
+				if len(iNum) <= vi {
 					return false
 				}
-				if len(jNum) < vi {
+				if len(jNum) <= vi {
 					return true
 				}
 				if iNum[vi] == jNum[vi] {

